berkeley-mobile/Utils/UIViewController+Extensions.swift

```diff
@@ -38,4 +40,42 @@ extension UIViewController {
         }
         self.present(alertController, animated: true, completion: nil)
     }
+    
+    // Presents an alert to open phone number
+    public func presentAlertLinkTelephone(title: String, message: String, options: String..., phoneNumber: String) {
+        let alertController = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        guard let number = URL(string: "tel://\(phoneNumber)") else { return }
+        
+        for (index, option) in options.enumerated() {
+            if (index == 0) {
+                alertController.addAction(UIAlertAction.init(title: option, style: .cancel))
+            } else {
+                alertController.addAction(UIAlertAction.init(title: option, style: .default, handler: { _ in
+                    UIApplication.shared.open(number, options: [:])
```

Comment:
Wonder if you should add a check using `UIApplication.shared.canOpenURL(_:)` before showing the alert or display some sort of response on failure. As is, if the number cannot be dialed (e.g. the user is on an iPad w/o a SIM or something) nothing happens after the user presses "Yes", which might be a bit confusing.